bin/build_container_image

```diff
@@ -2,13 +2,22 @@
 
 REGISTRY=${REGISTRY:-"docker.io"}
 ORGANIZATION=${ORGANIZATION:-"manageiq"}
+
+[[ -z "$TRAVIS_BRANCH" ]] && TRAVIS_BRANCH="$(git rev-parse --abbrev-ref HEAD)"
 DEFAULT_TAG="10"
 [[ $TRAVIS_BRANCH != "master" ]] && DEFAULT_TAG="latest-$TRAVIS_BRANCH"
 TAG=${TAG:-"$DEFAULT_TAG"}
+
 IMAGE_NAME=$REGISTRY/$ORGANIZATION/postgresql:$TAG
 
 set -e
 
+rm -rf images/postgresql-container
+git clone --branch generated --depth 1 https://github.com/sclorg/postgresql-container images/postgresql-container
```

Comment:
Maybe better to just pull a tarball, but we have git here, so this seemed easy enough